lib/msf/core/auxiliary/report.rb

```diff
@@ -303,7 +303,7 @@ def report_vuln(opts={})
 
     timestamp  = opts[:timestamp]
     username   = opts[:username]
-    mname      = self.fullname # use module name when reporting attempt for correlation
+    mname      = opts[:fullname] || self.fullname # use module name when reporting attempt for correlation
```

Comment:
Maybe:

```suggestion
    mname      = opts[:module_fullname] || self.fullname # use module name when reporting attempt for correlation
```

Reply:
That way it's consistent with existing APIs: https://github.com/search?q=repo%3Arapid7%2Fmetasploit-framework%20module_fullname&type=code

https://github.com/search?q=repo%3Arapid7%2Fmetasploit-credential%20module_fullname&type=code

https://github.com/search?q=repo%3Arapid7%2Fmetasploit_data_models%20module_fullname&type=code